### PR TITLE
Fix inconsistent help menu

### LINKED ISSUE #6368: Remove duplicated links from help menu
## Description

We want to simplify the help menu, because over the years, we've ended duplicating some of the links:

HELP
v4.8.0
[Documentation](https://documentation.wazuh.com/4.8)
[Community](https://wazuh.com/community/join-us-on-slack)
[Give feedback](https://wazuh.com/community/join-us-on-slack)
[Open an issue in GitHub](https://github.com/wazuh/wazuh/issues/new/choose)
Wazuh support
[ Documentation](https://documentation.wazuh.com/4.8/)
[ Slack channel](https://wazuh.com/community/join-us-on-slack)
[ Projects on Github](https://github.com/wazuh)
[ Google Group](https://groups.google.com/forum/#!forum/wazuh)

We want to simplify it to:

HELP
v4.8.0
[Documentation](https://documentation.wazuh.com/4.8)
[ Slack channel](https://wazuh.com/community/join-us-on-slack)
[ Projects on Github](https://github.com/wazuh)
[ Google Group](https://groups.google.com/forum/#!forum/wazuh)

The motivation for this changes are:
- Documentation will remain as it is, pointing users to our extensive documentation library
- Community is too broad to cover with a single link, so we want to provide links to our most used channels (slack and google groups), there are other channels like reddit and discord which are also popular.
- We want to link to the GitHub org so interested users can reach the all our open source projects

## Tasks

- [x] https://github.com/wazuh/wazuh-packages/pull/2796
- [x] https://github.com/wazuh/wazuh-dashboard-plugins/pull/6374
- [x] https://github.com/wazuh/wazuh-dashboard/pull/139

### PR DESCRIPTION
**Pull request by Tostti**
### Description

This PR modifies the help menu, to be updated to the current requirements, and to avoid the inconsistency and duplication that occured previously

### Issues Resolved

https://github.com/wazuh/wazuh-dashboard-plugins/issues/6368

### Evidence 
image
image

### Test
Verify that the help menu has the correct icons and links. 

**Note**: it is possible to run `yarn opensearch snapshot` in the repo before running `yarn start`, this will bring up an Opensearch instance that works with the dev environments. 

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
  - [ ] `yarn test:ftr`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/core/public/chrome/ui/header/header_help_menu.tsx [hunk 1]
```diff
@@ -198,37 +198,55 @@ class HeaderHelpMenuUI extends Component<Props, State> {
...
+      darkmode,
     } = this.props;
-    const { helpExtension, helpSupportUrl } = this.state;
+    const { helpExtension } = this.state;
 
     const defaultContent = useDefaultContent ? (
       <Fragment>
-        <EuiButtonEmpty href={opensearchDashboardsDocLink} target="_blank" size="xs" flush="left">
+        <EuiButtonEmpty
+          href="https://documentation.wazuh.com/4.9/"
```
Comment (yenienserrano) [inline src/core/public/chrome/ui/header/header_help_menu.tsx:209]: Perhaps the version could be dynamic to make it easier to maintain.

Comment (Tostti) [reply src/core/public/chrome/ui/header/header_help_menu.tsx]: Thanks, added on 3a26d552c763119ddde0346a5a9aa77150666838

**Review by lucianogorza — APPROVED** [review]
## CR 🟢 
## Test 🟢 
image

**Review by yenienserrano — APPROVED** [review]
CR: ✅ 
Test: ✅ 

image
image

**Pull request merged**